index.js

```diff
@@ -23,7 +23,8 @@ async function init() {
 	}
 
 	// A new tag must be created
-	const version = dailyVersion();
+	const prefix = core.getInput('prefix') || '';
+	const version = dailyVersion(prefix);
```

Comment:
```suggestion
	const version = dailyVersion(core.getInput('prefix'));
```

`core.getInput` returns an empty string anyway: https://github.com/actions/toolkit/blob/1cc56db0ff126f4d65aeb83798852e02a2c180c3/packages/core/src/core.ts#L76